src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicDoNotUseEnumerableMethodsOnIndexableCollectionsInsteadUseTheCollectionDirectlyFixer.vb

```diff
@@ -12,5 +14,15 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
         Private Protected Overrides Function AdjustSyntaxNode(syntaxNode As SyntaxNode) As SyntaxNode
             Return syntaxNode
         End Function
+
+        Private Protected Overrides Function IsConditionalAccess(syntaxNode As SyntaxNode) As Boolean
+            Return TypeOf syntaxNode Is ConditionalAccessExpressionSyntax
+        End Function
+
+        Private Protected Overrides Function ConditionalElementAccessExpression(expression As SyntaxNode, whenNotNull As SyntaxNode) As SyntaxNode
+            Dim arguments As IEnumerable(Of ArgumentSyntax) = {SyntaxFactory.SimpleArgument(DirectCast(whenNotNull, ExpressionSyntax))}
+            Dim argumentList = SyntaxFactory.ArgumentList(SyntaxFactory.SeparatedList(arguments))
+            Return SyntaxFactory.ConditionalAccessExpression(DirectCast(expression, ExpressionSyntax), SyntaxFactory.InvocationExpression(Nothing, argumentList))
```

Comment:
Is there any VB analog to C#'s `using static` to remove all `SyntaxFactory.` calls?

Reply:
@DoctorKrolic I guess `Imports Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory` should work